core-web/libs/edit-content/src/lib/fields/dot-edit-content-select-field/dot-edit-content-select-field.component.html

```diff
@@ -0,0 +1,6 @@
+<p-dropdown
+    [formControlName]="field.variable"
+    [options]="options"
+    [attr.aria-labelledby]="'aria-' + field.variable"
```

Comment:
Not sure if the word `aria` is a good idea for the id.

Reply:
This is for accesibility, i remove it for now, but we can check the a11y on primeng components later
https://primeng.org/dropdown

Reply:
Yes, we need this... but the `id="aria-..."` maybe we should use something like ``id="field-..."``